models/Asset/Service.php

```diff
@@ -493,4 +499,142 @@ public static function getUniqueKey(ElementInterface $element, int $nr = 0): str
 
         return $key;
     }
+
+    /**
+     * @throws \Exception
+     */
+    public static function getImageThumbnailByArrayConfig(array $assetInfo): null|ImageThumbnail|VideoImageThumbnail|DocumentImageThumbnail|array
```

Comment:
I know I am again a little nitpicky but should we maybe also call the parameter $config then instead of $assetInfo?
```suggestion
    public static function getImageThumbnailByArrayConfig(array $config): null|ImageThumbnail|VideoImageThumbnail|DocumentImageThumbnail|array
```

Reply:
Yes sorry about this. I was curious about this "Add suggestion to batch" :) 
Of course next time it will be a online. Sorry again!